### ISSUE TITLE
Executing Box 2 resulted in a warning.

### ISSUE THREAD

**Issue by tillerKleinmann** [zincware/ZnPINNLab#19]
[/home/tkleiner/Projects/uni/icp/pinn/day2/ZnPINNLab/day2/data_generator.py:117](https://file+.vscode-resource.vscode-cdn.net/home/tkleiner/Projects/uni/icp/pinn/day2/ZnPINNLab/day2/data_generator.py:117): UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  torch.tensor(psi(self.x)).float().to(device),
